tests/sparseml/transformers/finetune/finetune_custom/gpu/gpu_config.yaml

```diff
@@ -1,4 +1,4 @@
 cadence: "weekly"
 test_type: "regression"
-model: "zoo:llama2-7b-llama2_pretrain-base"
-file_extension: json
+model: "zoo:llama2-7b-ultrachat200k_llama2_pretrain-base"
+file_extension: json
```

Comment:
We should still specify either max_steps or num_train_epochs here, the default is 3 epochs which is way too high( we want 1 epoch)

Reply:
ah sorry I mean to ask you for an appropriate number of steps. Updated to just use num_train_epochs.

Nice catch!